caffe2/python/operator_test/reshape_ops_test.py

```diff
@@ -16,16 +16,16 @@ def test_lengths_to_shape_ops(self):
         workspace.RunOperatorOnce(core.CreateOperator(
             'LengthsToShape', ['l'], ['s']))
         workspace.FeedBlob('res', np.array([3, 200], dtype=np.int32))
-        assert ((workspace.FetchBlob('s') == workspace.FetchBlob('res')).all())
+        assert (workspace.FetchBlob('s') == workspace.FetchBlob('res')).all()
```

Comment:
Can we replace python assert with numpy assert_array_equal?

Reply:
Sure, will do after holidays :)